main/src/library/BigInt.flix

```diff
@@ -97,6 +97,116 @@ namespace BigInt {
         case _: ##java.lang.NumberFormatException => Err("BigInt.fromString")
     }
 
+    ///
+    /// Convert `x` to an `Option[Int8]`.
```

Comment:
F# favours using `try` as prefix for things it can't do with a total function and has to return an Option (e.g. tryHead). 

I'd be quite happy with this convention in "moderation" e.g. `head` and `last` in Flix's List module are safe versions that return `Option[a]` . I don't think they should be renamed to `tryHead` and `tryLast` but for these number conversions `tryToInt8` would be fine.

Reply:
I think this sounds like a good compromise.

Reply:
Good. I'll change the "Option" ones to `try_` when I add the `clamp_` functions.

Reply:
Name changes to use `try` prefix have been pushed.